akd_client/src/wasm.rs

```diff
@@ -137,10 +139,51 @@ pub fn lookup_verify(
     }
 }
 
+/// NOTE(new_config): Add a new configuration here
+
+/// Verify a lookup proof in WebAssembly for WhatsAppV1Configuration,
+/// utilizing serde serialized structure for the proof
+#[cfg(feature = "whatsapp_v1")]
```

Comment:
same all this can be cleaned up. I would still add a feature onto the `akd_client` crate, but it would just enable the downstream feature on `akd_core` without needing to be checked anywhere in this crate.